python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -1014,7 +1014,13 @@ def moveTable(self, table, new_table, new_schema=None):
         self._commit()
 
     def createView(self, view, query):
-        sql = "CREATE VIEW %s AS %s" % (self.quoteId(view), query)
+        user_input = view
+        
+        if '.' in user_input: # To allow view creation into specified schema
+            schema, view_name = user_input.split('.')
+            sql = "CREATE VIEW %s AS %s" % (self.quoteId([schema, view_name]), query)
+        else: # No schema specified; uses public
+            sql = "CREATE VIEW %s AS %s" % (self.quoteId(view), query)
```

Comment:
This should probably be hardened a little -- eg if the view name was entered badly with two periods, then we'll get a raw python exception here. How about:

```suggestion
        view_name_parts = view.split('.')
        if len(view_name_parts) == 2: # To allow view creation into specified schema
            schema, view_name = view_name_parts
            sql = "CREATE VIEW %s AS %s" % (self.quoteId([schema, view_name]), query)
        else: # No specific schema specified
            sql = "CREATE VIEW %s AS %s" % (self.quoteId(view), query)
```